src/Microsoft.ML.Core/Data/ModelLoading.cs

```diff
@@ -10,6 +10,11 @@
 
 namespace Microsoft.ML.Model
 {
+    /// <summary>
+    /// Signature for a repository based model loader. This is the dual of ICanSaveModel.
+    /// </summary>
+    public delegate void SignatureLoadModel(ModelLoadContext ctx);
```

Comment:
>public delegate void SignatureLoadModel [](start = 4, length = 39)

Maybe we can `[BestFriend] internal` this as we are with other delegates.

Reply:
Ok that makes sense

---
In reply to: [254473032](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254473032) [](ancestors = 254473032)